pages/common/groff.md

```diff
@@ -0,0 +1,24 @@
+# groff
+
+> Typesetting program that reads plain text mixed with formatting commands and produces formatted output.
+> It is the GNU replacement for the troff and nroff Unix commands for text formatting.
```

Comment:
Should we wrap troff and nroff with backticks ?

Reply:
Yes, good catch.